python/perspective/perspective/templates/bruce.html.jinja

```diff
@@ -0,0 +1,40 @@
+<script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective/dist/cdn/perspective.js"></script>
```

Comment:
Should we try to parameterize these to the same versions as used by the client?

Reply:
yeah for sure